src/main/java/org/takes/facets/cookies/RqCookies.java

```diff
@@ -78,7 +79,9 @@ public Base(final Request req) {
             throws IOException {
             final Map<String, String> map = this.map();
             final String value = map.getOrDefault(
-                new EnglishLowerCase(key.toString()).string(),
+                new UncheckedText(
+                    new Lowered(key.toString())
```

Comment:
@fanifieiev Isn't `key` already `String`? Why use `toString()` here?

Reply:
@fabriciofx Because key is CharSequence